docs/vcl.md

```diff
@@ -0,0 +1,80 @@
+## Introduction and Overview
```

Comment:
We can probably add one section at the bottom :

### known issues
- VCL currently only works for the primary interface when used in combination with multinet
- Dataplane restart is currently not detected by the client application, this can be alleviated by making healthchecks travel over the VCL.
- Source address spoofing is not supported even when disabling uRPF at the pod level.